docs/next/components/mdx/includes/dagster/DagsterDevTabs.mdx

```diff
@@ -0,0 +1,60 @@
+<TabGroup>
+<TabItem name="From a file">
+
+Dagster can load a file directly as a code location. In the following example, we used the `-f` argument to supply the name of the file:
+
+```shell
+dagster dev -f my_file.py
+```
+
+This command loads the definitions in `my_file.py` as a code location in the current Python environment
```

Comment:
both of these commands also let you include a "--working-directory" argument that determines the base folder for imports, not sure if we want to mention that here?

Reply:
I'm going to leave it out for now, but add it to the docs backlog.